frontend/models/product-list/index.ts

```diff
@@ -61,12 +61,11 @@ export async function findProductList(
    ]);
 
    const productList = result.productLists?.data?.[0]?.attributes;
-
    if (productList == null && deviceWiki == null) {
       return null;
    }
 
-   const deviceTitle = productList?.deviceTitle ?? filterDeviceTitle;
+   const deviceTitle = deviceWiki?.deviceTitle ?? productList?.deviceTitle;
```

Comment:
Does this need to be a string?
```suggestion
   const deviceTitle = deviceWiki?.deviceTitle ?? productList?.deviceTitle ?? '';
```

What was `filterDeviceTitle`?

Reply:
`filterDeviceTitle` is taken directly from the url. So for `/Parts/ibook` it would be `ibook`.

Reply:
We should probably default to string or null here: https://github.com/iFixit/react-commerce/blob/4b59f9c916a424b5e38b49f9a93781e4364ccf02/frontend/models/product-list/types.ts#L50

Reply:
Good call. I made it null.